examples/basics/plotting/mpl_plot.py

```diff
@@ -14,6 +14,7 @@
 import numpy as np
 import sys
 
+# import matplotlib.pyplot as plt
```

Comment:
Did you need to comment this out? Might as well delete it if we don't want it...